modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -111,25 +111,10 @@ definition {
 	test CanAddAccountEntryAfterCreatingAccountEntry {
 		property portal.acceptance = "true";
 
-		task ("Given a Account object and given an action to add a new Account entry after creating an Account entry") {
-			ObjectAdmin.addObjectViaAPI(
-				labelName = "Custom Object 180195",
-				objectName = "CustomObject180195",
-				pluralLabelName = "Custom Objects 180195");
-
-			ObjectAdmin.addObjectFieldViaAPI(
-				fieldBusinessType = "Integer",
-				fieldLabelName = "Custom Field",
-				fieldName = "customField",
-				fieldType = "Integer",
-				isRequired = "false",
-				objectName = "CustomObject180195");
-
-			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject180195");
-
+		task ("Given an Account object and given an action to add a new Account entry after creating an Account entry") {
```

Comment:
I have no idea what https://github.com/liferay-bpm-qa/liferay-portal/pull/753/commits/c73c8e01ce579ab311c2fea2925883df93f8c808 is doing, I can't review this commit since the changes are really hard to follow. It's better to push back code to the original author for them to make changes instead of making additional commits to fix their mistakes.

If you want to fix their mistakes, then you should do it this way:

1. Cherry pick first commit
2. Fix mistakes from the first commit in a new commit with a clear commit message explaining what you're fixing and why
3. Cherry pick second commit
4. Fix mistakes from the second commit in a new commit with a clear commit message explaining what you're fixing and why

This makes it a lot easier to understand what your changes are doing and makes it easier to review.